content/docs/welcome-guide/create/creating-projects-using-cli.md

```diff
@@ -113,9 +113,14 @@ If your project build directory is outside the project path, you must include th
         ```
 
         {{< important >}}
-If you built the engine from source using the `INSTALL` target, make sure that you launch the Editor and other tools from the install directory, _not_ the engine's build directory. The Windows install directory typically ends in `/bin/Windows/profile`.
+If you built the engine from source using the `INSTALL` target, make sure that you launch the Editor _and_ other tools from the installed engine's build directory, _not_ the engine's build directory. The Windows install directory typically ends in `/bin/Windows/profile`.
         {{< /important >}}
 
 You can also run Project Manager (`o3de.exe`) from the same directory to edit your project's settings, add or remove Gems from the project, rebuild your project, and launch the Editor.
 
+
+{{< caution >}}
+If you launch the Editor from the wrong directory, the **Asset Processor** processor from that directory will also launch.  You must close all **Asset Processor** tasks before you attempt to launch the Editor from a different directory.
```

Comment:
I think that use of the word "wrong" here might not be well-defined or even accurate in every case where this cautionary note is relevant. Instead, I think it's good as a general purpose note. For example, it's important to know when intentionally switching from one project to another.
```suggestion
When you launch the Editor, the **Asset Processor** from the same directory will also launch.  You must close all **Asset Processor** tasks before you attempt to launch the Editor from a different directory.
```